src/components/page-donate/page-donate.tsx

```diff
@@ -139,84 +139,122 @@ export class PageDonate {
+                    <p>Waiting in line sucks. Waiting in line with pizza sucks a little less.</p>
+                    <p>Keep our polling places joyful and welcoming places where no one has an empty stomach by chipping into the pizza fund today.</p>
+                  </div>
+
+                  <form id="donate-form" onChange={handleChange} onSubmit={handleCheckout}>
+                    <label class="label">
+                      Choose an amount <span class="required">*</span>
+                    </label>
+                    <ul class="donation-amount-list">
+                      <li>
+                        <label class="radio" htmlFor="radio-1">
+                          <input type="radio" value="20" id="radio-1" name="amount" />
+                          <span class="label-text">$20 🍕</span>
+                          <span class="indicator"></span>
+                        </label>
+                      </li>
+                      <li>
+                        <label class="radio" htmlFor="radio-2">
+                          <input type="radio" value="40" id="radio-2" name="amount" />
+                          <span class="label-text">$40 🍕🍕</span>
+                          <span class="indicator"></span>
+                        </label>
+                      </li>
+                      <li>
+                        <label class="radio" htmlFor="radio-3">
+                          <input type="radio" value="60" id="radio-3" name="amount" />
+                          <span class="label-text">$60 🍕🍕🍕</span>
+                          <span class="indicator"></span>
+                        </label>
+                      </li>
+                      <li>
+                        <label class="radio" htmlFor="radio-4">
+                          <input type="radio" value="100" id="radio-4" name="amount" />
+                          <span class="label-text">$100 🍕🍕🍕🍕🍕</span>
+                          <span class="indicator"></span>
+                        </label>
+                      </li>
+                      <li>
+                        <label class="radio" htmlFor="radio-5">
+                          <input type="radio" value="200" id="radio-5" name="amount" />
+                          <span class="label-text">$200 🍕🍕🍕🍕🍕🍕🍕🍕🍕🍕</span>
+                          <span class="indicator"></span>
+                        </label>
+                      </li>
+                      <li>
+                        <label class="radio" htmlFor="custom-amount">
+                          <span class="label-text">Other: $</span>
+                          <input class="input" type="text" name="amount" id="custom-amount" autocomplete="off" />
+                        </label>
+                      </li>
+                    </ul>
+                    {this.amount && !isNaN(this.amount) ? (
+                      <button onClick={handleCheckout} class="button is-red is-fullwidth-mobile">
```

Comment:
Can we make this full width for non-mobile?

![image](https://user-images.githubusercontent.com/700444/96624965-61480f80-12c2-11eb-8dfb-9dd48ee8be7d.png)

Reply:
Yep - I actually already pushed that change to this branch right after I requested the review. If you ran local, pull again (or refresh if on next.pizza)